cmd/collection.go

```diff
@@ -96,8 +96,37 @@ var listCollectionsCmd = &cobra.Command{
 
 var createCollectionCmd = &cobra.Command{
 	Use:   "collection {db} {schema}...|-",
-	Short: "create collection(s)",
-	Args:  cobra.MinimumNArgs(1),
+	Short: "Creates collection(s)",
+	Long: fmt.Sprintf(`Examples:
```

Comment:
There is special "Example" field in the cobra.Command, should we use it instead?

Reply:
May be it'll even be rendered nicely in the markdown.